assets/js/scripts.js

```diff
@@ -4,148 +4,161 @@
  * @package woo-address-book
  */
 
-jQuery( function ( $ ) {
+ var woo_address_book_app = {
 
-	var load_selectWoo = true;
-	var address_book = $( 'select#shipping_address_book:visible, select#billing_address_book:visible' );
+	shipping_address_from_cart: '',
 
-	// Check for Selectize being used.
-	if ( $.fn.selectize ) {
-		if ( address_book.hasClass( "selectized" ) && address_book[0] && address_book[0].selectize ) {
-			load_selectWoo = false;
+	init: function() {
+		s = this.settings;
```

Comment:
Is this `s = this.settings;` being using for anything?

Reply:
sorry, it wasn't my intention to add this. It was just for a test and I forgot removing it.

Reply:
Not a problem, just wanted to make sure I wasn't missing something where it was being used.